modules/users/manifests/maxlehmann.pp

```diff
@@ -0,0 +1,8 @@
+# Creates the maxlehmann user
+class users::maxlehmann {
+  govuk_user { 'maxlehmann':
+    fullname => Max Lehmann'',
```

Comment:
You've got a couple of rouge quotes here `''`.

Reply:
Thanks, now fixed.